### ISSUE TITLE
Create a link to embedded resources location in text from project homepage

### ISSUE THREAD

**Issue by krosemichael** [ManifoldScholar/manifold#2149]
## Feature Description

I would like to be able to see whether and where a resource is embedded in a text from the home page's activity feed and resource collection.

## *Why* is this feature important? *Who* does it help?

This feature would improve the utility of the home page by providing a quick route to seeing resources in context. It would help readers who were curious, from seeing a resource on the home page, about where it fits into the text. 

## User Stories

I am a reader who sees that an interesting image has been added to the project I am reading as a resource. I am curious whether the project creator has associated this image with a particular place in the text, but don't want to go through the project page by page to find out. I would like to be brought from the resource page to the place in the text where this resource has been embedded.

**Comment by zdavis** [comment]
How would we handle cases where a text had been annotated with a resource multiple times? What if a resource showed up in 10 different places in the text?

**Comment by krosemichael** [comment]
How about list of links to each annotation on the resource page? That could get clunky if there were a lot of resources that were annotated many times, but is that common?

**Comment by zdavis** [comment]
I'd argue pretty strongly against adding a list of links to a resource activity tile. Those tiles are meant to provide summary information about a resource that was added. To me, a list of links to the text feels like detail that's more appropriate on the resource page. 

While I see your point about multiple resource annotations not being common, from a development standpoint we have to design and build for a lot of edge cases that may not be common. If there was one resource annotation, a tasteful "view in text" link on the resource activity tile could work. If there were multiple, we'd probably have to take the reader to a disambiguation view that showed the annotations in context and helped the reader select which one he/she wanted to see. And then we'd also have to consider how to handle this content on the resource detail page. All of this is doable—it's just something that I'd want to do right and put through a bit of a design process.

All of which is to say—this feature would require UX design, so it's not a quick thing that we would just tackle in development.

**Comment by krosemichael** [comment]
Yes I agree a list of links to the annotations on the activity feed tile would be bad looking. I was suggesting it be on the resource page, but I see what you're saying in terms of design complications. Just to make sure I'm clear- my ideal flow here would be that the user clicks on the resource either from the activity feed or the resource collection, is taken to the resource page, and from there can access the links to the resource's location in the text if it's been embedded somewhere.

**Comment by zdavis** [comment]
Oh, in that case I don't have a problem with it—I thought you wanted them to click somewhere on the resource tile in the activity feed and go straight to the text. I can totally see this being added to the resource detail page, and think that's a great spot for it.

**Comment by zdavis** [comment]
@bot planned

**Comment by zdavis** [comment]
Thank you for taking the time to open this feature request. The Manifold team reviewed this issue during our bi-weekly meeting and the consensus is that this feature makes sense and is in keeping with our overall vision for the platform. Moreover, we see this request as a viable candidate for development under our current available funding. We’re adding a “planned” label to this feature to indicate that we plan on completing it within our current funding cycle.

_This was an automated message, but please don't hesitate to reply. Our team watches these issues closely and will respond as soon as we're able to!_
